packages/harpguru-core/src/components/menu-close-button/menu-close-button.tsx

```diff
@@ -1,15 +1,36 @@
-import { TapGestureHandler } from 'react-native-gesture-handler'
+import { useTimingTransition } from 'react-native-redash'
+import Animated, { Easing, interpolate } from 'react-native-reanimated'
+import { TapGestureHandler, State } from 'react-native-gesture-handler'
+import type { TapGestureHandlerStateChangeEvent } from 'react-native-gesture-handler'
 import { StyleSheet, View } from 'react-native'
 import React from 'react'
 import { AntDesign } from '@expo/vector-icons'
 
 import type { MenuProps } from '../../types'
 import { getSizes, colors } from '../../styles'
+import { tapAnimationDuration } from '../../constants'
 
 export const MenuCloseButton = ({
   openCloseTapHandler,
 }: Pick<MenuProps, 'openCloseTapHandler'>): React.ReactElement => {
   const sizes = getSizes()
+  const [isTapped, setIsTapped] = React.useState(false)
+  const transitionValue = useTimingTransition(isTapped, {
+    duration: tapAnimationDuration,
+    easing: Easing.inOut(Easing.circle),
+  })
+  const animationValue = interpolate(transitionValue, {
+    inputRange: [0, 1],
+    outputRange: isTapped ? [1, 0.5] : [1, 0.5],
+  })
+
+  const handleTapAndAnimate = ({
```

Comment:
The naming of all of the animation values and event handlers should follow a convention. I think it's getting a bit loose.